will_of_the_prophets/templates/will_of_the_prophets/public_board.html

```diff
@@ -2,26 +2,63 @@
 
 {% block content %}
 
-<div class="row pt-3">
-    <div class="col-lg-9">
-        <p class="h3 text-orange">The Greatest Generation Presents</p>
-        <h1 class="text-blue">Game of Buttholes:<br>The Will of the Prophets</h1>
+<h1 class="site__header">The Greatest Generation Presents:</h1>
+
+<section class="lcars-group">
+    <div class="lcars-group__horizontal-bar lcars-group__horizontal-bar--top">
+        <div class="lcars-group__filler lcars-group__filler--top-left-elbow lcars-group__filler--yellow">
+            GAME OF BUTTHOLES BOARD
+        </div>
+    </div>
+    <div class="lcars-group__left-bar">
+        <div class="lcars-group__filler lcars-group__filler--top-left lcars-group__filler--yellow">CODE 47</div>
+        <div class="lcars-group__filler lcars-group__filler--purple">ALL I DO IS BITS</div>
+        <div class="lcars-group__filler lcars-group__filler--lilac">MAIN COMPUTER CORE</div>
+        <div class="lcars-group__filler lcars-group__filler--bottom-left lcars-group__filler--yellow">DICK JOKES</div>
+    </div>
+    <div class="lcars-group__content">
+        <h2 class="lcars-group__content-header">
+            <span class="no-break">GAME OF BUTTHOLES</span> •
+            <span class="no-break">THE WILL OF THE PROPHETS</span>
+        </h2>
         {{ board }}
-        {% include "will_of_the_prophets/_footer.html" %}
     </div>
-    <aside class="col-lg-3">
-        <div class="row">
+    <div class="lcars-group__horizontal-bar lcars-group__horizontal-bar--bottom">
+        <div class="lcars-group__filler lcars-group__filler--bottom-left-elbow lcars-group__filler--yellow"></div>
+    </div>
+</section>
```

Comment:
Hey @craiga this lcars markup is to a point now where I think it might be useful to extract this out into a reusable template since I'm going to reuse it on the roll and roll_frequency pages, but I am unsure what the best approach is for that. The main things it needs to support are:
* Different content (inside of `lcars-group__content`).
* Dynamic colors and string contents inside of the different `lcars-group__filler` elements.

Should I just be using an include with some parameters that we can pass in? Or should I look at creating something that can be extended and then included here (so that we can use `{% block lcars_content %}` or something)? Or is there something else that I don't know about. It's been a few years since I worked with django.

Thanks!

Reply:
You probable want to use [include](https://docs.djangoproject.com/en/2.2/ref/templates/builtins/#include) for this.

```
{% include "lcars_thing.html" with color="yellow" content="all I do is bits" only %}
```

Reply:
Added an include that I'm now using all over the place. Thanks for the help!